packages/site/src/pages/visualTests/VisualTestGalleryPage.tsx

```diff
@@ -0,0 +1,107 @@
+import { Box, Gallery, Grid, Heading, Stack, Text } from "@jobber/components";
+import { useState } from "react";
+import { GalleryFile } from "@jobber/components/Gallery";
+
+export const VisualTestGalleryPage = () => {
+  const [files, setFiles] = useState<GalleryFile[]>([
+    {
+      key: "1",
+      name: "Image 1",
+      type: "image/jpeg",
+      size: 200000,
+      progress: 1,
+      thumbnailSrc: "https://placehold.co/200x200/e6e6e6/999999?text=1",
```

Comment:
I found that gallery failed once when I ran it caused by loading states. I think we could improve this by having the dev site locally provide these images instead of grabbing from a website that may have timing issues.

However, I am good to approve for now and we can fast follow this